src/cljs_test_runner/main.clj

```diff
@@ -106,6 +107,19 @@
     :else
     (edn/read-string (slurp path-or-data))))
 
+(defn add-loader-url
+  "Add url string or URL to the highest level DynamicClassLoader url set."
+  [url]
+  (let [u (if (string? url) (java.net.URL. url) url)
+        loader (loop [loader (.getContextClassLoader (Thread/currentThread))]
+                 (let [parent (.getParent loader)]
+                   (if (instance? DynamicClassLoader parent)
+                     (recur parent)
+                     loader)))]
+    (if (instance? DynamicClassLoader loader)
+      (.addURL ^DynamicClassLoader loader u)
+      (throw (IllegalAccessError. "Context classloader is not a DynamicClassLoader")))))
```

Comment:
Pretty cool stuff, will definitely be making use of this in other projects. Adding deps at runtime will be super handy with a Clojure plugin system I'll need soon. Thanks for a good example!